src/toolbox.js

```diff
@@ -12,14 +17,59 @@ var Toolbox = GObject.registerClass({
         props.orientation = Gtk.Orientation.VERTICAL;
         super._init(props);
 
-        const headerbar = new Gtk.HeaderBar({hasSubtitle: false});
-        const buttonReset = new Gtk.Button();
-        const image = new Gtk.Image({stock: 'gtk-revert-to-saved'});
-        buttonReset.add(image);
-        headerbar.pack_end(buttonReset);
-        headerbar.show_all();
-        this.add(headerbar);
+        this._headerbar = new Gtk.HeaderBar({
+            hasSubtitle: false,
+            title: this._title,
+        });
```

Comment:
Could be a bit cleaner to create this before chaining up, so you can remove the `if (this._headerbar)` check in the setter below?